### DISCUSSION TITLE
Possible to reduce space between horizontal MenuItem?

### DISCUSSION THREAD

**Discussion by michaelcsikos** [ant-design-blazor/ant-design-blazor#2805] [Q&A]
Is there an easy way to reduce the gaps between menu items with icons:

![image](https://user-images.githubusercontent.com/21806535/195782710-88486ecd-96c2-4db0-92fe-ae9d22f0c4bb.png)

**Comment by ElderJames** [comment]
Hi @michaelcsikos , maybe you need to use css to change the padding for menu items.
